firmware/nRF_badge/data_collector/incl/scanner.c

```diff
@@ -39,6 +39,58 @@ void scanner_init()
     //scan_state = SCAN_IDLE;
 }
 
+static ble_gap_evt_adv_report_t get_mock_adv_report_for_badge_with_id(uint8_t id) {
```

Comment:
This code should probably not be merged into master, but I'm keeping it on this branch for now so we can test tomorrow!